service/src/main/java/bio/terra/externalcreds/services/ProviderService.java

```diff
@@ -274,4 +288,275 @@ private void revokeAccessToken(
         linkedAccount.getProvider().toString(),
         responseBody);
   }
```

Comment:
everything below here copied verbatim from the now-deleted `PassportProviderService`